.NET/Microsoft.Recognizers.Text.DateTime/TasksModeConstants.cs

```diff
@@ -49,5 +49,39 @@ public static class TasksModeConstants
         // tasksmode specific date parser constant
         public const string NextWeekGroupName = "next week";
 
+        // constants
+        public const string PeriodDaySuffix = "P1D";
```

Comment:
Do we need to redefine many of these constants? It seems to me that many could reuse the standard one in Constants.

Reply:
Created a new variable in the TasksModeConstant file, with the intention of keeping the value specific to tasksmode if it is changed in the future.

Reply:
But for now, derived it's value from Constant File.